src/objects/agent/stats_manager.py

```diff
@@ -0,0 +1,14 @@
+AGENT_HP = 100
+
+
+class StatsManager:
+    def __init__(self, experience=0, hp=AGENT_HP) -> None:
+        self.experience = experience
+        self.hp = hp
+
+    def get_level(self) -> int:
```

Comment:
I think level of `Agent` should be an attribute of `StatsManager`, dynamically calculating it from other attribute is very opaque.